server-common/src/main/java/org/apache/cassandra/sidecar/db/schema/AbstractSchema.java

```diff
@@ -88,7 +88,7 @@ protected boolean initializeInternal(@NotNull Session session,
         }
 
         prepareStatements(session);
-        logger.debug("{} is initialized!", this.getClass().getSimpleName());
+        logger.info("{} table schema is initialized", this.getClass().getSimpleName());
```

Comment:
useful for debugging purposes. This gets logged a single time during startup and it produces useful information